crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -859,7 +859,7 @@ mod test {
                         current_number_of_peers: 1,
                         arc_size: 1.0,
                         total_network_peers: 1,
-                        bytes_since_last_time_queried: 1838,
+                        bytes_since_last_time_queried: 1915,
```

Comment:
I think the byte size increased here because the serialized bytes of entries is now larger as a result of the additional `tag = "type"` attribute.